examples/demo_opts.py

```diff
@@ -146,15 +146,15 @@ def get_device(actual_args=None):
             import luma.lcd.aux
             Device = getattr(luma.lcd.device, args.display)
             Serial = getattr(make_serial(args), args.interface)
-            luma.lcd.aux.backlight(bcm_LIGHT=args.bcm_backlight).enable(True)
+            luma.lcd.aux.backlight(bcm_LIGHT=args.gpio_backlight).enable(True)
```

Comment:
I'd also like to deprecate the `bcm` prefix in lcd (and lcd_matrix) so merging this PR is blocked till then.